commown_b2b_mail_channel/models/res_partner.py

```diff
@@ -13,3 +13,44 @@ class ResPartner(models.Model):
         string="Support mail chanel",
         domain=[("public", "=", "private"), ("channel_type", "=", "channel")],
     )
+
+    def write(self, vals):
+        """Override write function to add/remove company's partners when support channel is modified."""
+        if "mail_channel_id" in vals.keys():
+            new_chan_id = vals["mail_channel_id"]
+            if self.mail_channel_id and self.mail_channel_id.id != new_chan_id:
+                self.env["mail.channel.partner"].search(
+                    [
+                        ("channel_id", "=", self.mail_channel_id.id),
+                        ("partner_id", "in", self.child_ids.ids),
+                    ]
+                ).unlink()
+
+            if new_chan_id != False:
```

Comment:
`False` s'utilise avec `is`, même si ça marche, c'est pas très idiomatique. Ici je laisserais seulement `if new_chan_id:`.

Reply:
Fait